plugins/modules/sns_topic.py

```diff
@@ -153,6 +153,7 @@
 - amazon.aws.aws
 - amazon.aws.ec2
 - amazon.aws.boto3
+- amazon.aws.tags
 '''
 
 EXAMPLES = r"""
```

Comment:
Could you also please update documentation with the `tags` parameter? Please add also an example in the EXAMPLES block.